Makefile

```
@@ -16,11 +16,19 @@ build: $(BUF)
 test: lint build
 	go test -v ./...
 
+delete-generated:
```

Normally, this is called clean
```suggestion
clean:
```

---

There's another target called `clean` in Makefile.buf so they override each other if they're both named clean